ckanext/canada/helpers.py

```diff
@@ -175,6 +175,21 @@ def portal_url():
 
 def googleanalytics_id():
     return str(config.get('googleanalytics.id'))
+
+def adobe_analytics_login_required():
+    return "2" #return 1 if page requires a login and 2 if page is public
```

Comment:
Should there be a parameter here so we can determine whether a page is public or are you planning on using different helper functions for the portal+registry?

Reply:
Registry does not implement Adobe analytics at all, so it is not related to this function. It is only installed on the portal side. For now this function only returns 2 since all portal pages are public. But the function could be edited in the future if one day we end up having login required on the portal side.

Reply:
Can it though? With no parameters how would the helper know which page is being visited?

Reply:
Ok i see what you mean. For now I could add the url path or something, which will then be use in the logic once there is a need to implement it.